.github/workflows/ci.yml

```diff
@@ -49,14 +27,14 @@ jobs:
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
 
       - name: Set up environment
-        uses: mamba-org/setup-micromamba@f8b8a1e23a26f60a44c853292711bacfd3eac822 # v1.9.0
+        uses: mamba-org/setup-micromamba@617811f69075e3fd3ae68ca64220ad065877f246 # v2.0.0
         with:
           environment-file: environment.yml
           create-args: python=${{ matrix.python-version }}
           cache-environment: true
 
       - name: Install pip requirements
-        run: pip install ".[dev]"
+        run: pip install uv && uv pip install --system ".[dev]"
 
       - name: Run tests
-        run: nbdev_test
+        run: nbdev_test --do_print --timing --n_workers 0
```

Comment:
Oh, that is weird! I thought it would be `1` worker to run the unit tests sequentially with a single thread or process. Does `0` do the same thing here?

Reply:
0 means everything is run in the same (single) thread, see documentation of fastcore parallel